packages/abstract-provider/src.ts/index.ts

```diff
@@ -50,7 +50,8 @@ export type HederaTransactionRecord = {
     block_number?: number,
     hash: string,
     logs?: {},
-    accountAddress?: string
+    accountAddress?: string,
+    transfersList?: Array<{ to: string, amount: number }>
```

Comment:
I think that this transfer list will work **only** for `Hbar` transfers.
Will we ever support HTS (hedera token service) tokens?

Reply:
We won't support `HTS` tokens. You can resolve this.